app/assets/stylesheets/pages/my-notifications.scss

```diff
@@ -108,4 +111,7 @@
         @include font-size(16, 16);
     }
 
+    .link-container{
+        padding-top: 15px;
+    }
```

Comment:
Could you remove these three lines and add `margin-bottom:20px;` at the end of the `.no-notifications` styles above this? (This will make the margin/padding consistent between both pages)

Reply:
Also, could you add the following in this place (to style the button):
```css
.pure-button {
    background:$color-2;
    color:#fff;
    padding:10px 20px;
}
```